src/frontend/src/utils/http-request.util.ts

```diff
@@ -4,13 +4,31 @@ interface IHttpRequestOptions {
   headers: Record<string, string>;
   body?: Record<string, any>;
   additionalConfig?: Record<string, any>;
+  addLocalAuth?: boolean;
 }
 
 export class HttpRequest {
   private static async request<T>(options: IHttpRequestOptions): Promise<T> {
+    const requestHeaders: typeof options.headers = {
+      'Content-Type': 'application/json',
+    };
+
+    if (options.addLocalAuth) {
+      const accessToken = localStorage.getItem('access_token');
```

Comment:
in the util helper, there is a function named getItemFromStorage,maybe consider to use that one instead direct call localStorage

Reply:
The helper created is unused. It even parses the string data. We need to consider rewriting it and apply usage at all places @binggaofw. Since the storage is done directly (Screenshot below), we need unparsed string.
<img width="587" alt="image" src="https://user-images.githubusercontent.com/87394256/201791520-38a319b9-e770-43a0-a7c3-002e649084fc.png">

While I can update the helper and it's references, but looks like that can be for another day.  Thoughts?

Reply:
yes。 we can do it later